File: solutions/1. Two Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        lis=[]
        for i in range(len(nums)):
            check=0
           
            for j in range(i+1,len(nums)):
                if nums[i]+nums[j]==target:
                    check=1
                    lis.append(i)
                    lis.append(j)
                    break
            if check!=0:
                break
        return lis 
        

File: solutions/1. Two Sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int,int> st;
        vector<int> ans;
        for(int i=0;i<nums.size();i++){
            int s=target-nums[i];
            if(st.find(s)!=st.end()){
               ans.push_back(i);
                ans.push_back(st[s]);
                return ans;
            }
            st[nums[i]]=i;
        
    }
        return {};
    }
};
